pyproject.toml

```diff
@@ -29,6 +29,7 @@ gunicorn = "^19.9"
 psycopg2 = "^2.8"
 python-dateutil = "^2.8"
 pytz = "^2021.1"
+sentry-sdk = "^0.20.2"
```

Comment:
Please ensure the dependencies are sorted.

Reply:
Ummm. It is...